### PR TITLE
Update ckeditor.jsx

### LINKED ISSUE #46: Make sure editor instance is present when trying to destroy it
## Are you reporting a feature request or a bug?

Bug

## Provide detailed reproduction steps (if any)

See #16.

In some cases, component could be destroyed before editor initialization starts. The editor instance is not present and error will be thrown.

## Other details

* Browser: All
* OS: All
* Integration version: `1.0.0-beta` and below
* CKEditor version: -
* Installed CKEditor plugins: -

### PR DESCRIPTION
**Pull request by prokach**
Added check for editor before destroy it.

Closes #46.

### CODE REVIEW

File: src/ckeditor.jsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ class CKEditor extends React.Component {
 	}
 
 	_destroyEditor() {
-		this.editor.destroy();
+		this.editor && this.editor.destroy();
```
Comment (Comandeer) [inline src/ckeditor.jsx]: It would be more readable using traditional `if` clause:

```suggestion
		if ( this.editor ) {
			this.editor.destroy();
		}
```

**Comment by Comandeer** [comment]
I've added simple unit test to the change.

**Comment by Comandeer** [comment]
Rebased onto latest `master`.

**Comment by f1ames** [comment]
Rebased on the latest `master`.

**Pull request merged**